src/OptiHPLCHandler/empower_handler.py

```diff
@@ -173,6 +173,14 @@ def Status(self) -> List[HplcResult]:
         """Get the status of the HPLC."""
         raise NotImplementedError
 
+    def GetEmpowerProjects(self) -> list:
+        """
+        Assuming that the user has logged in in one project for example project = Mobile, this method fetches all available 
```

Comment:
Flake8 brokker sig over at linien er for lang (mere end 88 karakterer), or at den ender med et mellemrum

Line 179 ender også med et mellemrum

Reply:
ja jeg opdagede det først nu, og har i mellemtiden rettet det i min kode efterfulgt af en ny commit til git.